app/assets/v2/js/wallet.js

```diff
@@ -6,8 +6,8 @@ const eventDataWalletReady = new Event('dataWalletReady', {bubbles: true});
 if (!Object.hasOwnProperty.call(window, 'web3')) {
   window.web3 = null;
 }
-let web3Modal;
-let provider;
+window.web3Modal = null;
```

Comment:
Can you explain what's going on here? Having to make changes in this file is surprising to me.

Reply:
After reviewing the code again, these changes are not required any more. These changes where relevant for the initial attempts in creating a stub.
Will revert the changes to wallet.js